suites/quincy/cephfs/tier-3_cephfs_scale.yaml

```diff
@@ -164,3 +173,10 @@ tests:
       module: cephfs_scale.max_clones_from_snap.py
       name: "Clone scale testing"
       polarion-id: "CEPH-83575629"
+  - test:
```

Comment:
Can we remove the test section here ? As it is covered as part of the test CEPH-83581234. May be enhance the name to both vol and subvol?

Reply:
> Can we remove the test section here ? As it is covered as part of the test CEPH-83581234. May be enhance the name to both vol and subvol?

Neha the coverage here is for CephFS volume, what is mentioned at line 116 is the coverage for subvolume, i cant run them both back-to-back due to snap rotation issue we had discussed, i didnt have new polarion tc entry created here, neither i could have added same 83581234 entry,plz suggest

Reply:
As discussed, i have merged both tests and they will executed back-to-back but with interval for 10mins